templates/email/bexley/waste/other-reported.txt

```diff
@@ -1,9 +1,9 @@
 [% PROCESS 'waste/_email_data.html' ~%]
-Subject: [% IF is_missed_collection %]Thank you for reporting a missed collection[% ELSIF is_container_request %]Thank you for requesting a replacement bin[% ELSE %]Thank you for your [% report.category %][% END %]
+Subject: [% IF is_missed_collection %]Thank you for reporting a missed collection[% ELSIF is_container_request %]Thank you for requesting a replacement bin[% ELSIF is_request_enquiry %]Thank you for your replacement bin enquiry[% ELSE %]Thank you for your [% report.category %][% END %]
```

Comment:
Same here for the changes in this file, the ELSE clause now works (if you add a `| lower` to it which seems nice anyway).